console/src/components/UIKit/atoms/AlertBox/index.tsx

```diff
@@ -46,7 +51,7 @@ export const AlertBox = props => {
     >
       <Icon type={type} />
       {type && (
-        <Text as="span" pl="md" fontWeight="medium">
+        <Text pl="md" fontWeight="medium">
```

Comment:
Can you bring back `as="span"`? This is a part of the implementation.

Reply:
What will be the type of `as` prop,